src/beautifiers/sass-convert.coffee

```diff
@@ -0,0 +1,20 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class SassConvert extends Beautifier
+  name: "SassConvert"
+  link: "http://sass-lang.com/documentation/file.SASS_REFERENCE.html#syntax"
+
+  options:
+    # TODO: Add support for options
+    CSS: true
+    Sass: true
+    SCSS: true
```

Comment:
Instead of `true` (supports all options) it should be `false` (supports language with no options).

Resources
- https://github.com/Glavin001/atom-beautify/blob/master/docs/add-languages-and-beautifiers.md#how-to-add-a-beautifier-for-a-language